### DISCUSSION TITLE
Default PostgreSQL Character Set in Docs

### DISCUSSION THREAD

**Discussion by fabm3n** [pretix/pretix#2096] [Ideas] [answered]
I ran into the issue that my PostgreSQL Databse on Debian 10 was not UTF-8 by default.
I think the command to create the datbase in the docs should set UTF-8 as default for the pretix database.

How are your thoughts about this changes?

**Comment by fabm3n** [comment, marked as answer]
I just found out that my issue was using an LCX Container for testing where the charmap was not UTF-8 so PostgreSQL was not using UTF-8 as default.
Using a KVM virtual machine with UTF-8 as keymap and PostgreSQL is using UTF-8 as the default charset.

**Reply by raphaelm** [reply]
Great, but we could still add a note on this to the docs!

**Reply by fabm3n** [reply]
Yes, that's a good idea. But i will not make a PR. A CLA just for small changes in the Docs is a little bit too much :D

**Reply by fabm3n** [reply]
I will make a PR for this
